Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VREE.MSG

```diff
@@ -136,5 +134,4 @@
 {1104}{Vree304}{I don't know. Ask Manufacturing about it.}
 {1105}{Vree305}{He makes sure the Brotherhood stays on the path of righteousness.}
 {1106}{Vree306}{He enforces the High Elder's wishes. With force, if necessary.}
-{1107}{Vree200}{Please talk to one of the other Scribes. I'm trying to finish up reports for the other Elders.}

```

Comment:
All the keywords for the "Please talk to one of the other Scribes. I'm trying to finish up reports for the other Elders."-line are gone now?

Reply:
Yes, treat it now as an error for unknown TMAs.

Reply:
What do you mean with unknown TMAs? You deleted the keywords for that line. :D

Reply:
Didn't seem right to have the same message repeated many times, and because it's Vree's only TMA_MSG_DONTKNOW line. Optimization!